app/src/test/java/org/mozilla/tv/firefox/pocket/PocketVideoRepoTest.kt

```diff
@@ -7,10 +7,13 @@ package org.mozilla.tv.firefox.pocket
 import io.mockk.MockKAnnotations
```

Comment:
nit: I'd like to put your RCA & lessons learned in version control: GH may fail to exist while the repository remains.

There's a double-standard here – why don't we put everything in VC then? As a compromise for my time and readability, I think I try to put any results & conclusions into VC (even if it's duplicated) while the investigative details remain in the issues.